### DISCUSSION TITLE
BFF & VPS (replacing Telefunc RPC to REST API)

### DISCUSSION THREAD

**Discussion by oleghalin** [vikejs/vike#629] [Discussion] [answered]
Hello, thank you for awesome packages. 

Currently im using VPS & Telefunc together, but due to my team i want migrate all projects to same architecture with REST API. 

Currently we are introducing feature testing using MSW (https://mswjs.io/). So main reason why i want stop using telefunc is to be same between all projects with responses, communication between APIs etc. 

How can i achieve building backend endpoints with VPS. 

I digged a little bit and found honojs which looks very good, but i not sure that it makes sense to introduce another one router to VPS.

**Comment by brillout** [comment, marked as answer]
https://vite-plugin-ssr.com/api-routes
